draft-ietf-rtcweb-jsep.xml

```diff
@@ -2090,10 +2090,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                     candidates.</t>
 
                     <t>For DTLS, all m= sections MUST use the
-                    certificate for the identity that has been specified
-                    for the PeerConnection; as a result, they MUST all
-                    have the same <xref target="RFC4572"></xref>
-                    fingerprint value, or this value MUST be a
+                    certificate for the identity that has been
+                    specified for the PeerConnection; as a result,
+                    they MUST all have the same <xref
+                    target="I-D.ietf-mmusic-4572-update"></xref>
+                    fingerprint value(s), or this value MUST be a
```

Comment:
this value MUST be a session-level attribute -> these value(s) MUST be session-level attributes